scripts/generate/templates/Checkpoints.js

```diff
@@ -177,6 +195,13 @@ function length(${opts.historyTypeName} storage self) internal view returns (uin
     return self.${opts.checkpointFieldName}.length;
 }
 
+/**
+ * @dev Returns checkpoint at given position.
+ */
+function at(${opts.historyTypeName} storage self, uint32 pos) internal view returns (${opts.checkpointTypeName} memory) {
```

Comment:
I just want to acknowledge that this was essentially renamed from `getAtPosition` to `at` and that was somewhat opinionated but I'm okay with the change.

Reply:
The transition to clock() in Votes requires this function to be available in Trace224 and not just in History. So I aligned the naming wit other functions in that "common" template